source/code-snippets/usage-examples/deleteMany.js

```diff
@@ -1,21 +1,32 @@
+/* Delete multiple documents */
+
+// Import the MongoClient type from the mongodb package.
 import { MongoClient } from "mongodb";
 
 // Replace the uri string with your MongoDB deployment's connection string.
 const uri = "<connection string uri>";
 
+// Create a new client and connect to MongoDB.
 const client = new MongoClient(uri);
 
 async function run() {
   try {
+    // Access the movies collection from the sample_mflix database.
```

Comment:
Suggestion:

Lines 15-16 just get a reference to the collection. I think "access" means interact with it, which is started in line 22 when the deleteMany operation is called on the collection.